Master40.DB/Data/Context/MasterDBContext.cs

```diff
@@ -23,7 +23,7 @@ public class MasterDBContext : DbContext
         public DbSet<M_Stock> Stocks { get; set; }
         public DbSet<M_Unit> Units { get; set; }
         public DbSet<M_Operation> Operations { get; set; }
-        public DbSet<T_DemandToProvider> DemandToProviders { get; set; }
+        public DbSet<T_Demand> DemandToProviders { get; set; }
```

Comment:
Now, Provider and Demand Classes are separated but it should also split into 3 Tables for better Datahandling.
Demand
DemandToProvider
Provider